python/mrc/_pymrc/include/pymrc/segment.hpp

```diff
@@ -209,6 +209,14 @@ class BuilderProxy
                           std::shared_ptr<mrc::segment::ObjectProperties> source,
                           std::shared_ptr<mrc::segment::ObjectProperties> sink);
 
+    // TODO(devin) -- we can't pass all the variants, so we need to figure out how to do this
```

Comment:
Do we need to address this?

Reply:
For this, I was originally thinking about creating bindings that could take nodes and not just strings to look up nodes; at present, I don't think its worth the effort. There is going to be some additional work that needs to be done, at some point, to make creating splices using modules, from python, more user friendly. But its a fairly edge case issue, is will be non-breaking, and not required for this release.